training-tracker-service/src/main/java/org/tolking/controller/TrainingEventController.java

```diff
@@ -1,4 +1,4 @@
-package org.tolking.training_event_service.controller;
+package org.tolking.controller;
```

Comment:
initial package looked better, it described actual service. It's bad idea to have just generilized packages, since your libraries and other services will use same packages. It could create huge confusion during debugging, since you won't be able to easily find what class throwed exception. Moreover, there's a chance several services/libraries could have classes with same names and your code will become non-compilable

Reply:
I did it same package, because when we seend some dto through messaging, I also need to provide package and java file for consumer dto (in order to serialize and deserialize json)

Reply:
Do you have any alternnatives for this solution?

Reply:
I'd say serialize not with `@Serializable`, but just write json strings with `ObjectMapper`

Reply:
Or even better solution, extract your shared dtos to some library and reference it from your services.
But from my perspective, it you strongly don't have plans to read different types of messages from this queue (which, I think actually preferable way to build architecture), you can really just use json strings. https://stackoverflow.com/questions/71484819/spring-boot-jms-generic-json-messages-without-type-property